include/sys/fm/fs/zfs.h

```diff
@@ -57,9 +57,12 @@ extern "C" {
 #define	FM_EREPORT_PAYLOAD_ZFS_VDEV_GUID	"vdev_guid"
 #define	FM_EREPORT_PAYLOAD_ZFS_VDEV_TYPE	"vdev_type"
 #define	FM_EREPORT_PAYLOAD_ZFS_VDEV_PATH	"vdev_path"
+#define	FM_EREPORT_PAYLOAD_ZFS_VDEV_PHYSPATH	"vdev_physpath"
 #define	FM_EREPORT_PAYLOAD_ZFS_VDEV_DEVID	"vdev_devid"
 #define	FM_EREPORT_PAYLOAD_ZFS_VDEV_FRU		"vdev_fru"
 #define	FM_EREPORT_PAYLOAD_ZFS_VDEV_STATE	"vdev_state"
+#define	FM_EREPORT_PAYLOAD_ZFS_VDEV_STATE_STR	"vdev_state_str"
+#define	FM_EREPORT_PAYLOAD_ZFS_VDEV_PREVSTATE	"vdev_prevstate"
```

Comment:
It's a slippery slope but if we're adding a `vdev_state_str` then it's a little odd there's not a `vdev_prevstate_str`.